modules/installation-gcp-shared-vpc-ingress.adoc

```diff
@@ -0,0 +1,49 @@
+// File included in the following assemblies:
+// * installation/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-ingress_{context}"]
+= Optional: Adding ingress DNS records for shared VPC installations
+If the public DNS zone exists in a host project outside the project where you installed your cluster, you must manually create DNS records that point at the ingress load balancer. You can create either a wildcard `*.apps.{baseDomain}.` or specific records. You can use A, CNAME, and other records per your requirements.
+
+.Prerequisites
+* You completed the installation of {product-title} on GCP into a shared VPC.
+* Your public DNS zone exists in a host project separate from the service project that contains your cluster.
+
+.Procedure
+. Verify that the Ingress router has created a load balancer and populated the `EXTERNAL-IP` field by running the following command:
```

Comment:
There is inconsistency in the OCP docs with this, but according to Brandi Munilla, the word "Ingress" should always be capitalized. Above, you have "ingress load balancer." Here, you have "Ingress router." I'm inclined to default to Brandi and always opt to capitalize this word, but maybe we should opt to make it consistent one way or the other in your docs?

Reply:
I'm OK with capitalizing it in my docs. Thanks for pointing this out.